product-service/lambdas/createProduct.js

```diff
@@ -0,0 +1,100 @@
+import pg from 'pg';
+const { Client } = pg;
+
+const handleResponse = (products = {}, status = 200) => ({
+  headers: {
+    'Content-Type': 'application/json',
+    'Access-Control-Allow-Methods': '*',
+    'Access-Control-Allow-Origin': '*',
+  },
+  statusCode: status,
+  body: JSON.stringify(products),
+});
+
+const { PG_HOST, PG_PORT, PG_DATABASE, PG_USERNAME, PG_PASSWORD } = process.env;
+
+const credentials = {
+  user: PG_USERNAME,
+  host: PG_HOST,
+  database: PG_DATABASE,
+  password: PG_PASSWORD,
+  port: PG_PORT,
+  ssl: {
+    rejectUnauthorized: false,
+  },
+  connectionTimeoutMillis: 5000,
+};
+
+let data_export = {}, statusCode = 200;
+
+
+export const handler = async event => {
+  
+  console.log(event);
+
+  const client = new Client(credentials);
+  
+  client.on('error', err => {
+                            data_export = 'DB Client Error 500:' + err.stack;
+                            statusCode = 500;
+                            console.error('DB Client Error 500:', err.stack);
+                            })
+
+  await client
+          .connect()
+          .then(() => console.log('Client connected'))
+          .catch(err => {data_export = 'DB connection error:' + err.stack; statusCode = 500})
+
+
+  try {
+      
+      const { title, description, price, count } = JSON.parse(event.body);
+      
+      if (typeof title === 'undefined' || title === '') { data_export = 'Not valid data for product creation' ; statusCode = 400; return handleResponse(data_export, statusCode); }
+
+      await client.query('BEGIN');
+
+      const queryProduct = 'INSERT INTO products(title, description, price) VALUES($1, $2, $3) RETURNING id';
+
+      const valuesProduct = [title, description, price];
+
+      const queryStock = 'INSERT INTO stocks(product_id, count) VALUES($1, $2)';
+
+      const {rows: products} = await client.query(queryProduct, valuesProduct);
+
+      const productId = products[0].id;
+
+      const countStock = [productId, count];
+
+      await client.query(queryStock, countStock);
+    
+      console.log("Product created:" + productId);
+    
+      await client.query('COMMIT');
+    
+      data_export = {
+                    title,
+                    description,
+                    price,
+                    count,
+                    id: productId,
+                    };
+      
+      statusCode = 200;
+
+      } catch (err) {
+    
+      data_export = 'DB insert error 500:' + err; statusCode = 500;
```

Comment:
You should work a little with your code formatting. It's hard to read and review. It's just a good and useful habit - to keep everything neat and clean

Reply:
Thank you for your tip. And of course I agree with you.

But you should not check formatting of code. 
As the Task has no requirements on implementing Linter and Prettier, it should not be a point of your mark.

Reply:
No worries, it won't affect your score. It's not about task requirements, I always comment when I see something that could have been done better. And I always appreciate my colleagues' comments if they review my PRs